### DISCUSSION TITLE
Css navbar stylization

### DISCUSSION THREAD

**Discussion by DimasterGamer** [vercel/community#540] [Help] [answered]
I want to stylize my navigation bar, which is the best Css??

**Comment by swarnava** [comment, marked as answer]
Hi, We don't provide recommendation about specific CSS or JS framework. Let us know if you face any difficulty while deploying your project with Vercel and we will be more than happy to help! 😄

**Reply by DimasterGamer** [reply]
Okay, thanks for the help. I'm going to almost certainly need that help
